assets/js/googlesitekit/datastore/user/permissions.test.js

```diff
@@ -508,5 +509,78 @@ describe( 'core/user authentication', () => {
 				expect( canViewSharedAnalytics ).toBe( true );
 			} );
 		} );
+
+		describe( 'hasAccessToShareableModule', () => {
```

Comment:
We should add a test here to cover the "loading"/`undefined` state that could happen here.

Reply:
Added ✅